packages/tools/pactjs-cli/src/contract-generate/generate.ts

```diff
@@ -141,7 +141,10 @@ export const generate =
       if (existsSync(indexPath)) {
         console.log(`Appending to existing file ${indexPath}`);
         const indexDts: string = readFileSync(indexPath, 'utf8');
-        writeFileSync(indexPath, indexDts + exportStatement);
+        // Append the export to the file if it's not already there.
+        if (!indexDts.includes(exportStatement)) {
+          writeFileSync(indexPath, indexDts + exportStatement);
```

Comment:
Should we separate these with a newline or two (if they are not there?) Maybe something like

```
const separator = indexDts.endsWith('\n') ? '' : '\n\n';
const newIndexDts = [indexDts, exportStatement].join(separator);
writeFileSync(indexPath, newIndexDts);
```

Reply:
Agree, adding new line makes it more readable. 
maybe only adding one new line is enough. 
 ```javascript
  writeFileSync(indexPath, indexDts + exportStatement + "\n");
  ```

Reply:
My intent was for the newline to separate the inserted statement from the existing body - otherwise it would end up in the same line if the body does not end in a newline